oqs-template/generate-extras.yml

```diff
@@ -16,10 +8,6 @@ kem-extras:
     nid_ecx_hybrid: '0x2F42'
   frodo976shake:
     nid_ecx_hybrid: '0x2F43'
-  bike1l1cpa:
-    nid_ecx_hybrid: '0x2F46'
-  bike1l3cpa:
-    nid_ecx_hybrid: '0x2F47'
   kyber512:
     nid_ecx_hybrid: '0x2F26'
```

Comment:
Are you saying `nid_ecx_hybrid` for kyber512 already is in OQS-OSSL111 `generate.yml`? Shouldn't it then be completely dropped here? Feel free to change this file (commit to this branch) as you intended it to be (used): Maybe I didn't really understand it/the entries.

Reply:
yes, the nid_ecx_hybrid fields for kyber512, sikep434 and bike are already defined in OQS-OSSL111 generate.yml and yes, it would be cleaner to drop them from the generate-extras.yml. I will commit to this branch tomorrow with a proposed update.